jujugui/static/gui/src/app/components/user-profile/test-user-profile.js

```diff
@@ -584,95 +566,20 @@ describe('UserProfile', () => {
         charmstore={charmstore}
         env={env}
         getDiagramURL={sinon.stub()}
-        gisf={false}
         interactiveLogin={true}
         pluralize={sinon.stub()}
         storeUser={sinon.stub()}
         user={users.charmstore} />);
     assert.equal(list.callCount, 2);
   });
 
-  it('requests jem envs if jem is provided and updates state', () => {
```

Comment:
With this update the listModel is only provided a callback. The callback is called a passes through the correct data in this test: https://github.com/juju/juju-gui/pull/1668/files#diff-d56b204f2e7fc3ad8a502c26d7aa20fcR238